core/templates/components/forms/custom-forms-directives/image-uploader.directive.ts

```diff
@@ -51,18 +53,53 @@ angular.module('oppia').directive('imageUploader', [
             return 'This file is not recognized as an image.';
           }
 
-          if (!file.type.match('image/jpeg') &&
-            !file.type.match('image/gif') &&
-            !file.type.match('image/jpg') &&
-            !file.type.match('image/png') &&
-            !file.type.match('image/svg\\+xml')) {
+          var imageTypeMapping = {
+            jpeg: {
+              format: 'image/jpeg',
+              condition: [/jpeg$/, /\.jpeg$/],
```

Comment:
Where is the "condition" property used? Why does this not have `fileType` and `fileExtension` like the others in this dict?

Reply:
Also, have you tested image-uploader directive manually in other places where it is used with the file formats listed in this object?

Reply:
I am sorry for not noticing such a simple mistake, now I have fixed it. I had tested it in other places using png and not jpeg so I didn't realise the problem.

Reply:
Done.